### PR TITLE
List in alphabetical order

### LINKED ISSUE #66: Sort alphabetically?
Looking through the list, the order libraries are listed seems a bit random.
Would it make sense to go through the list and sort them by name?

### PR DESCRIPTION
**Pull request by thebuilder**
This fixes #66
Also align the naming to use kebab case

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,91 +25,91 @@ Awesome list of React components with render props and resources.
 
 ### Animation
 
-- [react-motion](https://github.com/chenglou/react-motion): A spring that solves your animation problems.
+- [react-morph](https://github.com/brunnolou/react-morph): Morphing Ui transitions made simple
 - [react-move](https://github.com/react-tools/react-move): Beautiful, data-driven animations for React
+- [react-motion](https://github.com/chenglou/react-motion): A spring that solves your animation problems.
```
Comment (ctrlplusb) [inline README.md:30]: `react-motion` should be above `react-move`

File: README.md [hunk 2]
```diff
@@ -25,91 +25,91 @@ Awesome list of React components with render props and resources.
...
-- [react-composer](https://github.com/jmeas/react-composer): Compose render prop components
-- [react-sentinel](https://github.com/YurkaninRyan/react-sentinel): Abstracts away requestAnimationFrame, allowing you to poll anything for props
-- [react-albus](https://github.com/americanexpress/react-albus): React component library for building declarative multi-step flows (wizards).
-- [react-pro-metronome](https://github.com/rigobauer/react-pro-metronome): React component for creating your own metronome
-- [react-with-async-fonts](https://github.com/sergeybekrin/react-with-async-fonts): Module for dealing with custom web fonts
-- [@tedconf/react-show-more](https://github.com/tedconf/react-show-more): A Unopinionated component to handle showing more items
-- [react-treefold](https://github.com/gnapse/react-treefold): A renderless tree component for your hierarchical React views
-- [@danwang/retainer](https://github.com/danwang/retainer): Compose render prop components by lifting computation into containers
+- [react-google-maps-loader](https://github.com/xuopled/react-google-maps-loader): Simple loader to use google maps services
```
Comment (ctrlplusb) [inline README.md:89]: `react-google-maps-loader` should be below `react-google-calendar-events-list`

File: README.md [hunk 3]
```diff
@@ -25,91 +25,91 @@ Awesome list of React components with render props and resources.
...
 - [react-sizeme](https://github.com/ctrlplusb/react-sizeme): Make your React Components aware of their width and height!
-- [react-intersection-observer](https://github.com/thebuilder/react-intersection-observer): Monitor when an element enters or leaves the browser viewport.
+- [@tedconf/react-show-more](https://github.com/tedconf/react-show-more): A Unopinionated component to handle showing more items
+- [react-t-minus](https://github.com/jxom/react-t-minus): A React component to handle simple countdowns.
+- [react-treefold](https://github.com/gnapse/react-treefold): A renderless tree component for your hierarchical React views
+- [react-useragent](https://github.com/jonstuebe/react-useragent): Retrieve user agent data through render props
+- [react-value](https://github.com/JedWatson/react-value): An easy easy way to wrap controlled components that provide `value` and `onChange` props with state.
+- [react-with-async-fonts](https://github.com/sergeybekrin/react-with-async-fonts): Module for dealing with custom web fonts
+- [@danwang/retainer](https://github.com/danwang/retainer): Compose render prop components by lifting computation into containers
```
Comment (ctrlplusb) [inline README.md:109]: `@danwang/retainer` should be below `render-props-compose`

**Review by ctrlplusb — CHANGES_REQUESTED** [review]
Thanks for getting this going @thebuilder - it definitely helps to have them sorted. 

I've noted a few places where the items were out of order.

A couple of comments from my side:
 
 - The whole "react-" prefix on libs throws me off a bit. I feel like it devalues the alphabetical ordering a bit. Not sure what to do here, just a note.
 - Should we consider moving scoped packages to the top of the lists? Seemed a bit strange having to read past the scoping, but maybe that is just me.

**Comment by ctrlplusb** [comment]
Side note: I am using your `react-intersection-observer` in a side project of mine. Really dig it, thanks for it! 😀

**Comment by thebuilder** [comment]
I agree about the `react-` prefix, but removing isn't really an option.
I think it would require rethinking how packages are presented.

### Ideas:
[Google Maps Loader](https://github.com/xuopled/react-google-maps-loader) *(react-google-maps-loader)* -  Simple loader to use google maps services
**Google Maps Loader** - [react-google-maps-loader](https://github.com/xuopled/react-google-maps-loader):  Simple loader to use google maps services

#### Table listing
Name | Package | Description
------------ | -------------  | -------------
Google Maps Loader | [react-google-maps-loader](https://github.com/xuopled/react-google-maps-loader)  | Simple loader to use google maps services

**Comment by ctrlplusb** [comment]
Name column is an interesting proposal, but not sure if it will always work. "Motion" vs "React Motion" for example. "React" is really a part of that libs name.  Maybe we just stick with as is for now and track this proposal separately.

**Comment by kentcdodds** [comment]
Yeah, I think the name column idea will be redundant most of the time. Also, going with a table layout will make this harder to contribute to, so let's just stick with the list.

**Comment by thebuilder** [comment]
Agreed.

**Review by ctrlplusb — APPROVED** [review]

**Comment by jaredpalmer** [comment]
While i like the visuals on the table, my only issue with a is that they can be very hard to edit on GitHub's online editor, making contribution a tougher.

**Pull request merged**